Makefile

```diff
@@ -1,8 +1,8 @@
 NAME          := hdf5-vol-daos
 SRC_EXT       := gz
 TEST_PACKAGES := daos-devel $(NAME) $(NAME)-mpich $(NAME)-openmpi3 $(NAME)-mpich-devel $(NAME)-openmpi3-devel $(NAME)-mpich-tests $(NAME)-openmpi3-tests
-SOURCE_COMMIT := b324b90d4554e211976174b9a3acdba61c184248
-TEST_COMMIT   := 31660ab19352049cf59f98b1fba498b3e93dade5
+SOURCE_COMMIT := 8d419310976bcd890521ebf72557325f34a8af56
+TEST_COMMIT   := d59811a39cd5053887f22cdabd1c08e6d9c539db
 SOURCE        = $(SOURCE_COMMIT).tar.$(SRC_EXT) 
 PATCHES       = $(TEST_COMMIT).tar.$(SRC_EXT)
```

Comment:
Are these two values used anywhere?  Should they be removed?  `$(PATCHES)` isn't really any patches yes?

Reply:
No they are not used and I removed them

Reply:
They are needed;  They override the SOURCE and PATCHES in Makefile_packaging.mk.  After I removed them the build failed with the following error


[2020-09-11T12:06:23.928Z] + rm -rf artifacts/centos7/

[2020-09-11T12:06:23.928Z] + mkdir -p artifacts/centos7/

[2020-09-11T12:06:23.928Z] + make CHROOT_NAME=epel-7-x86_64 chrootbuild

[2020-09-11T12:06:24.499Z] make: *** No rule to make target '%{source_commit}.tar.gz', needed by '_topdir/SOURCES/%{source_commit}.tar.gz'.  Stop.

Reply:
That seems like a deficiency in the packaging.  Probably the macros are not being defined in the spectool calls that assign the `SOURCES` and `PATCHES` values in the `packaging/Makefile_packaging.mk`.

Reply:
@mjean308 What about this?  It seems like it should be a pretty simple fix.  I really should have caught it on the PR it landed in. You should be able to just add `$(BUILD_DEFINES)` to the `spectool` arguments.